helm/values.yaml

```diff
@@ -21,9 +21,16 @@ service:
   port: 443
 
 # configure access to the notary server
-notaries:
-# public notary from dockerhub
-- name: dockerhub
+validators:
+- name: allow
+  type: static
+  approve: true
+- name: deny
+  type: static
+  approve: false
+- name: default
```

Comment:
I am not sure if this is the best way to do this from a user perspective to get started fast.
Shouldn't we give this configuration a specific name and use it in the image policy below explicitly for `docker.io/library` (if I am not mistaken, that is the docker official images). We could call this `docker_basics` and the docker key `docker_official`.
In that case, we could add a default config below that would be applied to all images that are not official or sse. Something like:
```
- name: default
  type: notaryv1
  host: notary.docker.io
  pub_root_keys:
  - name: default
    key: |
      -----BEGIN PUBLIC KEY-----
      -----END PUBLIC KEY-----
```
if this key is setup as the key for all non-specific images in the image policy, it would mean that a new user only has to add the public key and it would work out of the box with minimal changes. One thing, I would consider important, is that just cloning the repo and installing connaisseur w/o changes also works and gives a meaningful error message. Essentially, we would be validating against an empty key (or we add a random key in the default config that would have to be changed). What do you think?

Reply:
I like these ideas! However, I think we should not be putting random keys -- empty one (or placeholder) appeals much more to me. Otherwise, people don't know if they can change it or it's something hardcoded, and it looks suspicious anyway.

Reply:
I agree with the concerns regarding a random key. Would you think adding a place holder like `<add your public key here>` and then checking for the place holder during verification to give a reasonable error message like "no valid key specified for image" or something be okay?
Would achieve a super simple start without further actions, a simple adjustment small use cases by adding only one key and still be open to get the full functionality...

Reply:
Sure, why not. -- I think we should not have special treatment in our application code to check for special values like placeholders.